src/account-mgr.cpp

```diff
@@ -119,6 +120,8 @@ int AccountManager::saveAccount(const Account& account)
 
 int AccountManager::removeAccount(const Account& account)
 {
+    RepoService::instance()->onAccountRemoval(account);
+
```

Comment:
I did it for code simplicity when I wrote it. I saw the demands for the simplicity days ago. Have they changed yet?